src/core/reducers/api.js

```diff
@@ -74,6 +77,7 @@ export type ApiState = {
 export const initialApiState: ApiState = {
   clientApp: null,
   lang: null,
+  requestId: null,
```

Comment:
Passing the request ID to the API will be super easy now, thanks! :trop

Reply:
Oh, is that an upcoming feature? I'm not sure how helpful the request ID is on the client because it only correlates to the very first server response. So many things can happen after that!

Reply:
I'd like to explore this idea yeah, that would be nice to have correlations for everything happening on the server.

Reply:
I suppose it wouldn't hurt. I struggle to think of how it could be useful, though. A user could have many interactions with the API all tied to a single server response.